scripts/gitlab/test_linux_stable.sh

```diff
@@ -6,3 +6,5 @@ source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )/../com
 
 time cargo test --release --locked -p polkadot-node-core-dispute-coordinator --features disputes
 time cargo test --workspace --release --verbose --locked --features=runtime-benchmarks
+# Builds with the runtime metrics feature are only to be used for testing.
+time cargo test --workspace --release --verbose --locked --features=runtime-metrics
```

Comment:
can the feature be added to the previous command?

Reply:
see a comment above: https://github.com/paritytech/polkadot/pull/4483#discussion_r769590121